packaging_ignore.yml

```diff
@@ -9,6 +9,8 @@ base:
   - "gpg: WARNING: options in `/root/.gnupg/gpg.conf' are not yet active during this run"
   - ".* warning: overriding recipe for target 'check'"
   - "WARNING generated by debuild:"
+  - "Wrote: /citus-rpm-build/*"
+  - "\\d+ packages and \\d+ specfiles checked; \\d+ errors, \\d+ warnings."
```

Comment:
Shouldn't these be under rpm?

Reply:
There is an issue related to categorizing ignore under rpm category. Therefore, I added under general category